rele/subscription.py

```diff
@@ -30,14 +30,21 @@ def prefix(self):
     def set_prefix(self, prefix):
         self._prefix = prefix
 
+    @property
+    def filter_by(self):
+        return self._filter_by
+
+    def set_filter_by(self, filter_by):
+        self._filter_by = filter_by
+
     def __call__(self, data, **kwargs):
         if 'published_at' in kwargs:
             kwargs['published_at'] = float(kwargs['published_at'])
 
-        if self._filter_by(**kwargs):
-            return self._func(data, **kwargs)
+        if self._filter_by and not self._filter_by(**kwargs):
```

Comment:
Sorry, I don't understand what to expect from `self._filter_by(**kwargs)`

What do you think about extracting the condition to a meaningful method such `self._guard_against_processing_unfiltered_messages`?

Reply:
Yeah, good idea to extract it, I'm going to do that.